### PR TITLE
Make held suits slow you only for half

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

If certain suit has 60% speed modifier,  while held itll be only 80%
Only applied to suits tho, duffels still has all the same

## Why / Balance

current hardsuit slowdown, whether worn or held, feels overly punitive and unrealistic. Wearing a hardsuit should significantly reduce mobility due to its bulk, but merely holding it should result in a less severe encumbrance.

## Technical details

add new HalfSpeedModifier to HeldSpeedModifierComponent which checks in system and makes all the magic with modifiers

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Holding slowing down clothing now slows you by 50% of the slowdown when wearing them.

### CODE REVIEW

**Comment by Thurtik** [comment]
Realistically speaking, it would feel heavier in hand because when worn, the weight gets evenly spread across your whole body :)
But I do agree with this PR 👍

**Comment by Brandon-Huu** [comment]
Yeah, this makes no sense. 
It would be easier to move around while worn, not harder.

File: Resources/Prototypes/Entities/Clothing/Back/duffel.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@
     walkModifier: 1
     sprintModifier: 0.9
   - type: HeldSpeedModifier
+    halfSpeedModifier: true
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Clothing/Back/duffel.yml:17]: not necessary, given that this is the standard value in the component
```suggestion
```

Comment (lzk228) [reply Resources/Prototypes/Entities/Clothing/Back/duffel.yml:17]: oh i was about to set false here, thanks for notice

**Comment by metalgearsloth** [comment]
Was discussed among maints tl;dr

image

**Pull request closed without merging**